.expeditor/chef_server.sh

```diff
@@ -6,7 +6,7 @@
 
 git clone https://github.com/chef/automate.git
 cd automate
-git checkout kalroy/cs_plan_changes
+git checkout ssudheer/ruby_update_3.1
```

Comment:
Make it configurable but for now it's ok.

Reply:
Made it configurable.